tests/Application/src/Entity/LegacyBook.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\Component\Resource\Model\TranslatableInterface;
+use Sylius\Component\Resource\Model\TranslatableTrait;
+
+#[ORM\Entity]
+#[ORM\MappedSuperclass]
+#[ORM\Table(name: 'app_legacy_book')]
+class LegacyBook implements ResourceInterface, TranslatableInterface
+{
+    use TranslatableTrait {
+        __construct as private initializeTranslationsCollection;
+    }
+
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private int $id;
+
+    public function getId(): int
```

Comment:
```suggestion
    private ?int $id = null;

    public function getId(): ?int
```

Current typehint will throw exception on call on new entity.

Reply:
Would be cool to sync code with regular Book/BookTranslation

> tests/Application/src/Entity/Book.php